Assets/Source/Components/UI/DashCooldownBehavior.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Assets.Source.Components.Actor;
+using Assets.Source.Components.UI.Base;
+using UnityEngine.UI;
+
+namespace Assets.Source.Components.UI
+{
+    public class DashCooldownBehavior : HUDBase
```

Comment:
Nitpicky kinda, but can this class's name be updated to reflect that its a hud element?  (not actual cooldown logic)

Reply:
i can try but i thought it was already pretty accurate since its displaying the current dash cooldown remaining